### DISCUSSION TITLE
How to deploy a Monai code through monai-deploy sdk?

### DISCUSSION THREAD

**Discussion by Bhagii2508** [Project-MONAI/MONAI#4237] [Q&A]
Hello,
I have a monai segmentation code and want to deploy it. I have gone through the tutorial video of monai deploy SDK, it only talks about utilizing the deploy SDK by  developing application within the deploy file. I have already a perfectly running code file and I just want to deploy it without happening to write all the code again in the deploy SDK file.

Thank you

**Comment by wyli** [comment]
Thanks for starting this discussion, I think there is a plan to make "auto-deploy" via `monai.bundle` which is a new feature for the upcoming release. https://github.com/Project-MONAI/MONAI/discussions/4219 we'll get back to this topic when it's in a better shape.
